operator/src/main/java/oracle/kubernetes/weblogic/domain/model/DomainSpec.java

```diff
@@ -112,10 +113,11 @@ public class DomainSpec extends BaseConfiguration {
    *   BY_SERVERS - log files are organized under loghome/servers/server name/logs.
    * */
   @Description(
-      "Control how the log files under logHome is organized. "
-        + "FLAT - all files are under the logHome root directory. "
-        + "BY_SERVERS (default) - domain log file and introspector.out are at the logHome root level, all other files "
-        + "are organized under the respective server name logs directory.  logHome/servers/<server name>/logs.")
+      "Control how log files under `logHome` are organized when logHome is set and `logHomeEnabled` is true. "
+        + "`FLAT` - - all files directly in the `logHome` root directory. "
```

Comment:
Second logHome is missing back-ticks.

Maybe change "- -" to "-".

directory. --> directory (remove period)

logHome/servers/<server name>/logs is missing back-ticks

Reply:
fixed